snapshot_settings.json

```diff
@@ -10,22 +10,42 @@
     "defaultTab": "all"
   },
   "network": "1",
+  "voting": {
+    "type": "basic",
+    "quorum": 75000,
+    "delay": 0,
+    "period": 604800,
+    "hideAbstain": false
+  },
   "plugins": {
-    "gnosis": {},
     "safeSnap": {
-      "address": "0x0eBaC21F7f6A6599B5fa5f57Baaa974ADFEC4613"
-    }
+      "safes": [
+        {
+          "network": "1",
+          "realityAddress": "???",
+          "multisend": "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761"
```

Comment:
This config is currently not supported. The multisend address is hardcoded to `"0x8D29bE29923b68abfDD21e541b9374737B49cdAD"` in SafeSnape.

Reply:
Oh weird, for some reason I thought we added support for the space to define their multisend address. @carlosfebres, did add support for the space to define their multisend address?

Reply:
Ah right, ignore my comment please, I was looking at an outdated version of the SafeSnap code 🤦‍♂️